CMake/dashboard-scripts/KWIVER_common.cmake

```diff
@@ -372,7 +372,7 @@ while(NOT dashboard_done)
       if(COMMAND dashboard_hook_submit)
         dashboard_hook_submit()
       endif()
-      ctest_submit()
+      ctest_submit(FILES "${KWIVER_BINARY_DIR}/nose_results.xml")
```

Comment:
Isn't this line already in the 'finish_bindings.." PR? Assuming the intent is for this to land then to rebase that one?

Reply:
Exactly right.